security_privacy_ln.asciidoc

```diff
@@ -0,0 +1,614 @@
+[[security_and_privacy_of_ln]]
+== Security and Privacy of the Lightning Network
+In this chapter, we look at the issues related to the security and privacy of the LN.
+
+= Why is privacy important? =
+
+The key value proposition of cryptocurrency is censorship resistant money.
+It has been shown many times that the power of governments over money is often abused.
+Bitcoin offers individuals the possibility of storing and transferring their wealth without restrictions.
+
+Lightning continues this mission.
+It aims at providing a better user experience with instant payments while not sacrificing censorship resistance.
```

Comment:
I think this sentence is a bit too strong to present without first attempting to define what is meant by "censorship resistance" here. As it's a layer two system, it makes certain tradeoffs at a fundamental level in favor of faster lower cost payments compared to the based layer. As a result, it makes addition assumptions which may be characterized as it having less "censorship resistance" compared to the base layer.

Reply:
```suggestion
Unlike trivial scaling solutions: centralized Bitcoin banks, LN aims to scale Bitcoin without compromising on self-custody, which leads to greater censorship resistance in the Bitcoin ecosystem.
```